docs/user-documentation/soc-and-feeds.md

```diff
@@ -113,7 +113,36 @@ In Swarm, `topic` is a 32-byte long arbitrary byte array. It's possible to choos
 const topic = bee.makeFeedTopic('my-dapp.eth/outbox')
 ```
 
-### Reading feeds
+### High level JSON API
+
+High level JSON API is a convenience API that is bound to JSON data types. It allows easily setting and getting of a value to/from feed.
+
+:::info Bee's instance signer
+You can pass a [`Signer`](../api/types/signer.md) (or compatible data) into [`Bee` class constructor](../api/classes/bee.md#constructor), which then
+will be used as default `Signer`.
+:::
+
+```js
+await bee.setJsonFeed(
+  'some cool arbitraty topic', 
+  { some: ['cool', { json: 'compatible' }, 'object']}, 
+  { signer: 'some private key or Signer object' }
```

Comment:
I wouldn't show a syntactically correct example if that does not work. It would be better to either make a working example (that may be difficult in this case because providing a random private key may give a wrong impression on how they are supposed to use) or a non-sensical one, so that it's clear to the user that they need to figure out the correct form on their own.

Reply:
Hmm, I don't completely agree with you as I think they would understand it but I have changed it to use the same signer as your `makeFeedWriter` one.